iphone/metadata/en-US/release_notes.txt

```diff
@@ -1,27 +1,34 @@
-We and our contributors have prepared several Christmas/New Year presents for you:
+We have prepared several Christmas/New Year presents for you:
+
 • Outdoors map style for hiking, cycling, and exploring the Nature
-• Search for addr:place village addresses without streets in Austria, Czechia, Germany, Poland, Slovakia
+• Search for village addresses without streets in Austria, Czechia, Germany, Poland, Slovakia and some other countries
 • Type "skiing" or "ski run" in the search to find downhill and Nordic pistes
 • OpenStreetMap map data as of December 13
 • Added a button to see photos, reviews, and prices for some hotels. Every booking made through the app contributes to the development of the project!
 
 Other notable fixes and improvements:
 • Fixed several crashes and freezing on iOS 12, 13, 14
 • Multiple KML files can be imported from a single KMZ file
-• A credit card is displayed for places with ATM
-• A dollar emoji is displayed where a fee is requred
+• Less "gaps" in highways on the World map
+• Removed very short ferry lines from the World map
+• A credit card is displayed for selected places with ATM
+• A dollar emoji is displayed where a fee is required
 • Prettified websites in Place Page
 • "Zoom in to see isolines" message does not cover buttons
-• CarPlay search button now properly shows search results
+• CarPlay Search button now properly shows search results
 • Improved user interface for the Search on iPad
-• Added Singapore metro icon and fixed Porto metro icon
-• Added Hackerspace
-• Fixed leisure garden areas
-• Consistent displaying of house numbers on zoom 17
+
+Styles
+• Singapore metro icon
+• Hackerspace
+• Fix Porto metro icon
+• Fixed residential leisure garden area fills disappearing on some zoom levels
+• Display house numbers regardless of buildings' sizes
```

Comment:
"on z17"
on z16 the building size logic is used still
though I understand that this "zXX" notion is not clear to the users (and maybe the whole concept of descreet zoom levels)...